src/haggadah/events.cljs

```diff
@@ -87,6 +86,15 @@
                      :on-success (keyword->func on-success)
                      :on-error (keyword->func on-error)}}))))
 
+(re-frame/reg-event-fx
+ ::fetch-sedarim
+ (fn [{:keys [db]} [_ {:keys [on-success on-error] :or {on-error :error}}]]
+   (if (:uid db)
```

Comment:
In which case we do a `fetch-sedarim` without a user?

Reply:
You're right. The sedarim are only brought when we have a user. I'll remove the `if` wrapping the event.

Reply:
I was reviewing the code, and saw that the seradim are fetched before the user's information is obtained. The if is necessary to avoid querying with an invalid path.

Reply:
perhaps should be chained somehow... wait for the user to be fetched and then do more stuff

Reply:
I'll add an issue for having the user fetched first before doing anything else.